.ci.yaml

```diff
@@ -1455,6 +1475,18 @@ targets:
       benchmark: "true"
     scheduler: luci
 
+  - name: Linux_samsung_s10 backdrop_filter_perf__timeline_summary
+    recipe: devicelab/devicelab_drone
+    bringup: true # https://github.com/flutter/flutter/issues/92612
+    presubmit: false
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab","android","linux","samsung"]
```

Comment:
Should the s10 not also be appended to the tags?

Reply:
It would be a nice to have.  We now use this target tags only for statistic analysis. The `tags` here is different from the `Skia tags`, which are being collected in recipes side. One related cl: https://flutter-review.googlesource.com/c/recipes/+/27361.

Added for better info collection.

Reply:
What are the SkiaPerf tags for these benchmarks? They should also be more specific that just "samsung" for the same reason.

Reply:
Here is a [sample led run](https://luci-milo.appspot.com/raw/build/logs.chromium.org/flutter/led/keyonghan_google.com/0aaacd86f0f7266587fa34465311554f175ffd7619b77f5562bac7e2f9085bfa/+/build.proto) to show the tags: 
```
{"arch": "intel", "device_type": "**SM-G973U1**", "device_version": "android-30", "host_type": "linux", "host_version": "debian-10.11"}
```

We are using `'adb shell getprop ro.product.model' to tell the device type for all android devices.